airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -10,12 +10,14 @@
 
 import pendulum as pdm
 import requests
+from airbyte_cdk.sources.streams.http import HttpClient
 from requests.exceptions import JSONDecodeError
 from source_shopify.utils import ApiTypeEnum
 from source_shopify.utils import ShopifyRateLimiter as limiter
 
+from ...http_request import ShopifyErrorHandler
```

Comment:
Is there a reason why we use relative import here instead of `source_shopify.http_request`?

Reply:
Who should answer this question?

Reply:
I don't know but I haven't encountered any issue associated to this. It just felt odd that we do that here while we use fully qualified import elsewhere

Reply:
Currently, there is no such import (for master), i believe this one was added by you, with this change, if not, please attach the related PR that implemented this line, because i'm confused)